src/data/cloud-computing-quiz.ts

```diff
@@ -1,4 +1,13 @@
 const cloudComputingQuiz = [
+  {
+    Question: "Which cloud computing delivery model focuses on providing infrastructure like servers, networking technology, storage, and data center space as a service to users?",
+    Answer: "IaaS",
+    Distractor1: "FaaS",
+    Distractor2: "Saas",
+    Distractor3: "PaaS",
+    Explanation: "IaaS cloud computing delivery model focuses on providing infrastructure like servers, networking technology, storage, and data center space as a service to users. This gives users the autonomy to decide what infrastructure is provisioned based on the different needs of their application.",
+    Link: "https://www.freecodecamp.org/news/advantages-of-cloud-computing/"
```

Comment:
this article covers IaaS

```suggestion
    Link: "https://www.freecodecamp.org/news/get-started-with-cloud-native/"
```